README.md

```diff
@@ -124,6 +124,7 @@ Set the variable ``PMA_ABSOLUTE_URI`` to the fully-qualified path (``https://pma
 * ``PMA_PORTS`` -  define comma separated list of ports of the MySQL servers
 * ``PMA_USER`` and ``PMA_PASSWORD`` - define username to use for config authentication method
 * ``PMA_ABSOLUTE_URI`` - define user-facing URI
+* ``HIDE_PHP_VERSION`` - if defined, will hide the php version (`expose_php = Off`)
```

Comment:
From my understanding and testing, using `-e HIDE_PHP_VERSION` is not sufficient to trigger this; the variable must be set to something (anything). Therefore, I suggest improving the documentation about this to show how exactly a user can use this flag on the command line.

Having no better idea, I'd say this could say ```if defined, will hide the php version (`expose_php = Off`). Set to any value (such as HIDE_PHP_VERSION=true).```

Reply:
I applied your change because it looks good and clear